GenderPayGap.WebUI/Views/Register/ReviewAddress.cshtml

```diff
@@ -102,43 +119,128 @@
+                            </th>
+                            <td class="govuk-table__cell">
+                                @Model.OtherValue
+                            </td>
+                        </tr>
+                    }
                 </tbody>
             </table>
-
-            <fieldset>
-                <br>
-                <p>
-                    <button type="submit" class="button" name="command" value="approve">Confirm and complete registration</button>
-                </p>
-                <p>
-                    <button type="submit" class="link-button" name="command" value="decline">Reject registration</button>
-                </p>
-            </fieldset>
-        </div>
+            
+        <h2 class="govuk-heading-l">
+            Similar organisations
+        </h2>
+        
+        @if (Model.ManualEmployers != null && Model.ManualEmployers.Any()) {
+            
+            <table class="govuk-table" summary="A list organisations that match your organisation details">
+                <caption class="govuk-table__caption govuk-visually-hidden">Customer and Business listing</caption>
+                <thead class="govuk-table__head">
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="col">Organisation and address</th>
+                        <th class="govuk-table__header" scope="col">References</th>
+                        <th class="govuk-table__header" scope="col">Registration Status</th>
+                    </tr>
+                </thead>
+                <tbody class="govuk-table__body">
+                    @for (var i = 0; i < Model.ManualEmployers.Count; i++)
+                    {
+                        <tr class="govuk-table__row">
+                            <td class="govuk-table__cell">
+                                @Model.ManualEmployers[i].OrganisationName <span>@Model.ManualEmployers[i].GetFullAddress()</span>
+                            </td>
+                            <td class="govuk-table__cell">
+                                @{ Dictionary<string, string> references = Model.GetReferences(i); }
+                                @foreach (string key in references.Keys)
+                                {
+                                    if (key.EqualsI("Company No") && references[key].EqualsI(Model.CompanyNumber)
+                                        || key.EqualsI("Charity No") && references[key].EqualsI(Model.CharityNumber)
+                                        || key.EqualsI("Mutual No") && references[key].EqualsI(Model.MutualNumber)
+                                        || key.EqualsI(Model.OtherName) && references[key].EqualsI(Model.OtherValue)
+                                        )
+                                    {
+                                        <span style="color: red; display: block; white-space: nowrap;">@key: @(references[key])</span>

```

Comment:
I do not like inline css. Inline css I do not like

Reply:
I can see the Dulcie copied this from another page.

I'm not sure of the best way to manage CSS that's:
* from old-style pages
* that we want to re-write

We could include it in the new-style Sass/CSS structure, but it would be nice to have a way to make it clear that we don't "approve" of the overall CSS structure of the page, we just want to move some of it out of inline CSS - maybe just some comments?